isis/tests/CMakeLists.txt

```diff
@@ -8,11 +8,41 @@ file(GLOB test_source "${CMAKE_SOURCE_DIR}/tests/*.cpp")
 file(GLOB isis_libs "${CMAKE_BINARY_DIR}/lib/*.so")
 file(GLOB isis_mac_libs "${CMAKE_BINARY_DIR}/lib/*.dylib")
 
+set(MISSION_LIBS
```

Comment:
:+1: Huge improvement, thank you. Will need to remember to update this list if a new mission gets added unless this is updated again in the future to get a list of the current missions.